src/gc-arena/src/lib.rs

```diff
@@ -26,15 +23,18 @@ mod unsize;
 #[doc(hidden)]
 pub use unsize::__CoercePtrInternal;
 
+pub mod cell;
+
+pub type GcCell<'gc, T> = Gc<'gc, cell::RefCell<T>>;
+pub type GcWeakCell<'gc, T> = Gc<'gc, cell::RefCell<T>>;
```

Comment:
This should be using GcWeak I believe
```suggestion
pub type GcWeakCell<'gc, T> = GcWeak<'gc, cell::RefCell<T>>;
```

Reply:
Yeah, you're right :c